internal/build/boil_runs.go

```diff
@@ -7,24 +7,20 @@ import (
 
 func BoilRuns(runs []model.Run, pathMappings []PathMapping) ([]model.Cmd, error) {
 	res := []model.Cmd{}
+	localPaths := PathMappingsToLocalPaths(pathMappings)
 	for _, run := range runs {
-		if run.Triggers == nil {
+		if run.Triggers.Empty() {
 			res = append(res, run.Cmd)
 			continue
 		}
-		matcher, err := ignore.CreateRunMatcher(run)
+
+		anyMatch, err := ignore.AnyMatchGlobs(localPaths, run.Triggers)
```

Comment:
i feel like the existing `ignore` package tends to use the terminology "Match" and "Pattern" a lot. The "glob" terminology seems new? it would be nice if this integrated better with the existing naming conventions